pom.xml

```diff
@@ -95,32 +95,16 @@
             <version>3.6.28</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.sourcelab</groupId>
+            <artifactId>kafka-connect-client</artifactId>
+            <version>3.1.0</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>com.github.spotbugs</groupId>
```

Comment:
did you mean to remove spotbugs?

Reply:
no, must've accidentally removed this in pom updates